Gemfile.lock

```diff
@@ -1000,6 +1013,7 @@ DEPENDENCIES
   timecop (~> 0.9)
   twilio-ruby (~> 5.31)
   twitter (~> 6.2)
+  typhoeus (~> 1.3.1)
```

Comment:
Rails projects have such a funny tendency to end up with multiple HTTP clients installed 😃 Technically any client with keep-alive and a Faraday adapter should work (see [comment here](https://github.com/elastic/elasticsearch-ruby/issues/41#issuecomment-37272954)), but probably best to stick to one of the two recommend ones.

Reply:
we'll consolidate eventually :D 

judging from the `Gemfile.lock` I see at least `faraday`, `rest-client`, `httparty`, `httpclient`, `net-http-persistent` and now `typhoeus`

Reply:
Haha, yes, pretty much every Rails project I saw ended up like this. Not a big deal, but I always find it amusing.